src/sentry/api/endpoints/organization_details.py

```diff
@@ -191,4 +186,10 @@ def delete(self, request, organization):
                 transaction_id=transaction_id,
             )
 
+            delete_logger.info('object.delete.queued', extra={
+                'object_id': organization.id,
+                'transaction_id': transaction_id,
+                'model': type(organization).__name__,
```

Comment:
To pass the work off to `create_audit_entry` https://github.com/getsentry/sentry/pull/4022/files#diff-ec9caea86e4f8f68c1e2ab06547f2d7eR114